### PR TITLE
Do a full Java configuration (no xml)

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: fullstack/pom.xml [hunk 1]
```diff
@@ -24,6 +24,7 @@
         <awaitility.version>1.7.0</awaitility.version>
         <commons-io.version>2.5</commons-io.version>
         <commons-lang.version>3.4</commons-lang.version>
+        <ehcache3.version>3.2.0</ehcache3.version>
```
Comment (anthonydahanne) [inline fullstack/pom.xml]: is it overriding a spring boot property ?
why isn't spring boot using this version in the first place ?

Comment (henri-tremblay) [inline fullstack/pom.xml]: Yes. I know. They haven't upgraded. You should probably submit a PR

**Comment by henri-tremblay** [comment]
Do merge yet. I will do the clustered version to have everything clean

File: fullstack/pom.xml (hunk 1 above)
Comment (anthonydahanne) [inline fullstack/pom.xml]: let's wait for spring boot 1.5.0 to be released : https://github.com/spring-projects/spring-boot/blob/1.5.x/spring-boot-dependencies/pom.xml#L74

**Comment by henri-tremblay** [comment]
Ok. So clustering is now supported

**Pull request merged**